web/src/components/SelectionDialog.js

```diff
@@ -21,7 +22,19 @@ class SelectionDialog extends React.Component {
     this.setState({ open: true })
   }
 
+  getMirandaURL() {
+    var url = urljoin('https://github.com/Wattpad/highlander/tree/master/k8s/charts/services', this.props.serviceName)
```

Comment:
We should be able to configure the base urls for our "source code repo" and "chart repo", without having them specifically reference highlander and miranda.